frontend/common/code-help/traits/traits-rest.js

```diff
@@ -0,0 +1,23 @@
+module.exports = (envId, { FEATURE_NAME, FEATURE_FUNCTION, USER_ID, FEATURE_NAME_ALT }) => `# This will create a user in the dashboard if they don't already exist
+
+curl -X "POST" "https://edge.api.flagsmith.com/api/v1/identities/?identifier=winning2"
```

Comment:
The query param here is redundant (I think the FE just belt and braces it but let's be succinct here and remove it).